src/main/java/uk/gov/digital/ho/hocs/casework/api/BankHolidayResource.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.casework.domain.repository.BankHolidayRepository;
+
+@RestController
+public class BankHolidayResource {
+    BankHolidayService bankHolidayService;
+
+    @Autowired
+    public BankHolidayResource(BankHolidayService bankHolidayService) {
+        this.bankHolidayService = bankHolidayService;
+    }
+
+    @GetMapping(value = "/bankHoliday/refresh")
+    public ResponseEntity<String> refreshFromApi() {
+        int holidaysAdded = bankHolidayService.refreshBankHolidayTable();
```

Comment:
Is this return useful? Where would we use the number of holidays added? 200 status be enough to just identify success?